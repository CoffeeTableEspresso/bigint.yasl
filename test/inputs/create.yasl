
const bigint = require('bigint')

const n = bigint.bigint('123456789123456789')

echo n

const k = bigint.bigint(1000)

echo k

echo bigint.bigint(k)
