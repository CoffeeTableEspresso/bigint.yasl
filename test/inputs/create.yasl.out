123456789123456789
1000
1000
